src/Operations.jl

```diff
@@ -425,7 +425,9 @@ function deps_graph(ctx::Context, uuid_to_name::Dict{UUID,String}, reqs::Require
     for uuid in uuids
         uuid == uuid_julia && continue
         if !haskey(uuid_to_name, uuid)
-            uuid_to_name[uuid] = registered_name(ctx, uuid)
+            name = registered_name(ctx, uuid)
+            name === nothing && pkgerror("cannot find name corresponding to UUID $(uuid)")
```

Comment:
I feel we should do more to emphisize the seriousness of this error.
Since it is not something someone should be expected to resolve on their own, or by reading docs.

Reply:
maybe we should have `criticalpkgerror` for this kinda thing?

Reply:
I'm not sure why this error would be more serious than any other error.

Reply:
```
(12) pkg> test
ERROR: trying to test unnamed project
```

is something the user can fix.
It is a user error.

This on the other hand,  it took you and I over an hour.
And you know a whole lot more about the package system internals than the average user could ever be expected to.

Reply:
In this case it was a broken install, but this can happen in normal use also, e.g. if you send me a project + manifest with some private package. I just don't see how the extra `"this is serious"` helps anything, that doesn't give the user any more information. I'll add something like
```
"cannot find name corresponding to UUID $(uuid) in a registry"
```

Reply:
Sounds good to me